sklearn/datasets/base.py

```diff
@@ -352,8 +352,10 @@ def load_iris(return_X_y=False):
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
-        meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        meaning of the features, 'DESCR', the full description of
+        the dataset, 'filename', the physical location of iris csv dataset.
+
+        .. versionadded:: 0.20
```

Comment:
That seems a bit unclear. data was not added. Maybe specify that 'filename' was added. Same for other datasets.

Reply:
@amueller 
Will something like "the full description of the dataset, 'filename', the physical location of iris csv dataset **(added in version `0.20`)**" do the job ?

I don't get how to use `..versionadded:: 0.20` to specify the version for the attribute only :/